client/task-list/tasks/tax.js

```diff
@@ -167,6 +167,21 @@ class Tax extends Component {
 		);
 	}
 
+	doNotChargeSalesTax() {
+		const { updateOptions } = this.props;
+
+		queueRecordEvent( 'tasklist_tax_connect_store', {
+			connect: false,
+			no_tax: true,
+		} );
+
+		updateOptions( {
+			woocommerce_no_sales_tax: true,
```

Comment:
Looks like we should also set `woocommerce_calc_taxes` to `no` in this scenario.

Reply:
Nice, done in 266be839